iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -710,12 +710,6 @@ void ConvertToGPUPass::runOnFunction() {
     PartitionPLoopToWorkgroups, RemoveLinalgRange>(context);
   // clang-format on
 
-  patterns.insert<MapLinalgOpToGlobalInvocationId<linalg::ConvOp>>(context);
-  if (useLegacyConvLowering)
```

Comment:
Is it also possible to eliminate the command line option now?

Reply:
Yes, I will do it. Its WIP. I created a PR just to import it into internal google tooling to check some previously failing internal tests. Those are still failing (cant repro it anywhere else though). I will have to wait for Hanhan's tooling.